Misc/NEWS.d/next/Documentation/2021-03-04-04-55-16.bpo-43319.weh_qw.rst

```diff
@@ -0,0 +1 @@
+fixed the tutorial on venv about standard library
```

Comment:
I don't think this is needed. I've added the `skip-news` label, so not having a `NEWS` entry shouldn't be a problem.

Reply:
Yes, I'm not familiar with it and added it by accident, I will drop it.